strip_solutions.py

```diff
@@ -60,6 +60,18 @@ def clean(filepath, destination):
             "*.ipynb_checkpoints",
             "README.md",
             "LICENSE",
+            "references.bib",
+            "*.html",
+            "*.yml",
+            "_static",
+            "article",
+            "clean_metadata.py",
+            "glossary.md",
+            "favicon.ico",
+            "intro.md",
+            "requirements.txt",
+            "strip_solutions.py",
+            "update_workbook.py",
```

Comment:
Why do you copy the scripts over?

Reply:
I think I'm ignoring them?

Reply:
Yes, never mind! `ignore` was outside the diff shown by GH and I assumed this was an include list, not exclude.